Sources/Danger/Plugins/SwiftLint/SwiftLint.swift

```diff
@@ -169,14 +217,11 @@ extension SwiftLint {
         }
     }
 
-    static func swiftlintDefaultPath(packagePath: String = "Package.swift") -> String {
-        let swiftPackageDepPattern = "\\.package\\(.*SwiftLint.*"
-        if let packageContent = try? String(contentsOfFile: packagePath),
-            let regex = try? NSRegularExpression(pattern: swiftPackageDepPattern, options: .allowCommentsAndWhitespace),
-            regex.firstMatchingString(in: packageContent) != nil {
-            return "swift run swiftlint"
+    private static var tmpSwiftflintOutputFilePath: String {
+        if #available(OSX 10.12, *) {
+            return FileManager.default.temporaryDirectory.appendingPathComponent("swiftlintReport.json").path
         } else {
-            return "swiftlint"
+            return try! FileManager.default.url(for: .cachesDirectory, in: .userDomainMask, appropriateFor: nil, create: true).appendingPathComponent("swiftlintReport.json").path
```

Comment:
<!--
  1 failure:  Force tries shoul...
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/Plugins/SwiftLint/SwiftLint.swift;
  Line: 224;
-->
- :no_entry_sign: Force tries should be avoided. (`force_try`)
- :warning: Line should be 120 characters or less: currently 178 characters (`line_length`)